phpunit.xml.dist

```diff
@@ -18,6 +18,12 @@
             <directory>./tests/</directory>
         </testsuite>
     </testsuites>
+    <logging>
+        <log type="coverage-html" target="./tests/report" charset="UTF-8"
+            yui="true" highlight="true"
+            lowUpperBound="50" highLowerBound="80"/>
+        <log type="testdox-html" target="./tests/report/testdox.html" />
+    </logging>
     <filter>
         <whitelist>
             <directory>./system/</directory>
```

Comment:
Change this to ./application/
I think we want report on the application folder instead of the system folder

Reply:
done